controller/instance_manager_controller.go

```diff
@@ -149,6 +149,11 @@ func NewInstanceManagerController(
 	}, 0)
 	imc.cacheSyncs = append(imc.cacheSyncs, ds.KubeNodeInformer.HasSynced)
 
+	ds.SettingInformer.AddEventHandlerWithResyncPeriod(cache.ResourceEventHandlerFuncs{
```

Comment:
NIT: you can use `AddEventHandlerWithResyncPeriod` instead. See the node controller as an example.

Reply:
Updated.